v03_pipeline/lib/reference_data/compare_globals_test.py

```diff
@@ -113,6 +114,37 @@ def test_create_globals_from_dataset_ht_configs(
             },
         )
 
+    @mock.patch.dict('v03_pipeline.lib.reference_data.compare_globals.CONFIG', CONFIG)
+    @mock.patch(
+        'v03_pipeline.lib.reference_data.dataset_table_operations.hl.read_table',
+    )
+    def test_create_globals_from_dataset_configs_single_dataset(self, mock_read_table):
+        mock_read_table.return_value = B_TABLE
```

Comment:
This test case mocks `hl.read_table` instead of `import_ht_from_config_path` so we have coverage of that function being called. (I couldn't do this in the first test in this file because dataset 'a' has a custom_import,  so I had to mock the call to `import_ht_from_config_path`). This test also covers the case of using the `single_dataset` param.